src/test/java/com/github/mauricioaniche/ck/integration/IntegrationTest.java

```diff
@@ -54,7 +54,40 @@ public void notifyError(String sourceFilePath, Exception e) {
         });
     }
 
+    @Test
+    void testIndividualFiles() throws GitAPIException {
+        clone("https://github.com/apache/commons-lang.git");
```

Comment:
Can this test run in one of the fixtures we have? This way we avoid cloning a very large project, and things would go faster!

Reply:
I have changed the structure somewhat such that the tests are still separate so you see separate failures but the clone happens only once. Also I made it so the ck code does not need to be written twice.